src/Exceptions/SPLUnexpectedValueException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace League\CLImate\Exceptions;
+
+class SPLUnexpectedValueException extends \UnexpectedValueException implements Exception
```

Comment:
What's the purpose of this class? Shouldn't the `UnexpectedValueException` class just extend `\UnexpectedValueException`?

Reply:
In some places of old code `\UnexpectedValueException` was used instead of just `\Exception`. I've created this class ti follow backward compatibility in case some one catches `\UnexpectedValueException` class, but added or own child of it.